server/middlewares/rateLimit.middleware.js

```diff
@@ -4,8 +4,8 @@
 const ipchecker = require('ipchecker')
 const defaults = {
   duration: 1000 * 60 * 60,
-  whiteList: [],
-  blackList: [],
+  allowList: [],
+  blockList: [],
```

Comment:
Are these consumer-facing options? Do we need backwards compatibility? @pastelsky